PNA/agilentN5242A.py

```diff
@@ -304,7 +326,8 @@ def freq_sweep_phase(self, c, log=False):
         sweeptime, npoints = yield self.startSweep(dev, sweepType)
         if sweeptime > 1:
             sweeptime *= self.sweepFactor(c)
-            yield util.wakeupCall(2*sweeptime)  #needs factor of 2 since it runs both forward and backward 
+            # needs factor of 2 since it runs both forward and backward
```

Comment:
I didn't write these comments. I think removing them is out of bounds of this PR.